### DISCUSSION TITLE
Unable to see any of my devices on iOS 17

### DISCUSSION THREAD

**Discussion by galalmounir** [nVuln/homebridge-lg-thinq#266] [Q&A]
I see my devices fine on homebridge UI, and I can interact with them. But they don't show up on the Home app on any of my devices (running iOS 17 on all)

**Comment by nVuln** [comment]
please remove homebridge out of home app first, reset all homebridge cached accessories, then re-add it to home app again
let see if it's can be fixed

**Comment by yjeanrenaud** [comment]
Any updates here?

**Reply by nVuln** [reply]
you need manual add it in Home app, go to Add Accessory in home app and you can see your device there
you can check homebridge log for pin code

**Reply by yjeanrenaud** [reply]
Yes but that does not work

**Reply by yjeanrenaud** [reply]
fyi, I had  [homebridge-gsh](https://github.com/oznu/homebridge-gsh) removed and re-installed. Now the LG device shows up in homekit. So it likely had nothing to do with this plugin
